docs/development.md

```diff
@@ -0,0 +1,28 @@
+# River
+
+River is an experimental Postgres queue for Go.
+
+## Development
+
+### Run tests
+
+Raise test databases:
+
+    go run ./internal/cmd/testdbman create
+
+Run tests:
+
+    go test ./...
```

Comment:
Unfortunately our current `-p 1` requirement makes this not quite right 😞

Reply:
Heh, true. Existing README problem unfortunately.